atomic_reactor/tasks/sources.py

```diff
@@ -45,29 +45,22 @@ def prepare_workflow(self) -> inner.DockerBuildWorkflow:
 class SourceBuildTask(SourceBuildBaseTask):
     """Source container build phases task."""
 
-    plugins_def = plugin_based.PluginsDef(
-        prebuild=[
-            {"name": "fetch_sources"},
-            {"name": "bump_release"},
-        ],
-        buildstep=[
-            {"name": "source_container"},
-        ],
-        postbuild=[
-            {"name": "tag_and_push"},
-            {"name": "verify_media", "required": False},
-            {"name": "koji_import_source_container"},
-            {"name": "koji_tag_build"},
-        ],
-    )
+    plugins_conf = [
+        {"name": "fetch_sources"},
+        {"name": "bump_release"},
+        {"name": "source_container"},
```

Comment:
Now that we don't have the build phase I think this should be `build_source_container` since just `source_container` donesn't make any sense

Reply:
I'm not sure if it's necessary to rename it and even yes, rename are also involves renaming other things like the plugin class name. I'm going to keep it here in this pr. The rename can be done separately.

Reply:
I would say this PR is a more appropriate place to do the rename rather than having PRs for non-functional changes but I'll leave that up to you.